apps/asap-server/test/integration/fetch-users.integration-test.ts

```diff
@@ -51,4 +51,77 @@ describe('Users', () => {
       items: [expect.objectContaining(expectedResponse)],
     });
   });
+
+  test('Invalid orcids should fail', async () => {
+    const randomEmail = chance.email();
+    const randomName = chance.guid();
+
+    // Refactor duplicate user like above?
+    const user: Partial<User> = {
+      firstName: 'John',
+      lastName: randomName,
+      jobTitle: 'Project Manager',
+      orcid: 'invalid orcid',
+      institution: 'Instituto Superior Tecnico',
+      email: randomEmail,
+      role: 'Grantee' as Role,
+      degree: 'MPH',
+    };
+
+    try {
+      const result = await createUser(user);
```

Comment:
im not a huge fan of try catching inside tests. Could you try and use the jest `expect(...).rejects.toThrow()` syntax?